modulefiles/gsi_common.lua

```diff
@@ -30,5 +31,7 @@ load(pathJoin("nemsio", nemsio_ver))
 load(pathJoin("wrf_io", wrf_io_ver))
 load(pathJoin("ncio", ncio_ver))
 load(pathJoin("crtm", crtm_ver))
-load(pathJoin("ncdiag",ncdiag_ver))
+--load(pathJoin("ncdiag",ncdiag_ver))
 
+setenv("ncdiag_ROOT", "/scratch1/NESDIS/nesdis-rdo2/David.Huber/GSI/ncdiag/1.1.1")
+setenv("ncdiag_VERSION", "1.1.1")
```

Comment:
Lines 36-37 need to be removed.

Reply:
This will be covered by hu5970/GSI#12.  This is the branch I am working with to perform testing on Jet, Gaea, and Cheyenne.